src/util/logger.hpp

```diff
@@ -1,19 +1,14 @@
-#include <utility>
-
-/*!
- * \author ddubois
- * \date 21-Aug-18.
- */
-
-#ifndef NOVA_RENDERER_LOGGER_HPP
-#define NOVA_RENDERER_LOGGER_HPP
+#pragma once
 
 #include <functional>
 #include <mutex>
 #include <sstream>
 #include <string>
 #include <unordered_map>
+#include <utility>
 
+// Hope this fixes windows build for now, not sure where it comes from
+// TODO: Fix the root of this issue (@dethraid, @cwfitzgerald)
```

Comment:
The root of the issue is that windows.h is stupid. We need to undefine ERROR

Reply:
Please change these comments to something like "windows.h defines ERROR, and Nova uses ERROR as a lot level, so we need to undefine ERROR so we can it"